builder/BUILD

```diff
@@ -64,15 +76,28 @@ cc_test(
 cc_test(
     name = "pipeline_test",
     srcs = ["pipeline_test.cc"],
+    data = ["replica_pipeline_srcs.cc"],
```

Comment:
I'd just call this `replica_pipeline_test.cc`

Reply:
It's not really a test, but ok

Reply:
IMO it should be though, and I made this comment before seeing that isn't the case. That's what I was imagining when I suggested this. Sorry it wasn't more clear. The alternative that you implemented had not occurred to me as a possibility. I elaborated more in a later comment.